plugins/github_changeset/app/models/samson/github/changeset.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+module Samson
+  module Github
+    class Changeset
+      attr_reader :repo, :previous_commit, :commit
+      BRANCH_TAGS = ["master", "develop"].freeze
+      ATTRIBUTE_TABS = %w[files commits pull_requests risks jira_issues].freeze
```

Comment:
this is the same / needs to stay the same ... so how about we don't duplicate them ?